src/get.rs

```diff
@@ -153,16 +153,26 @@ pub unsafe extern "C" fn z_get(
     let mut get = session.get(Selector::new(key_expr, p));
     if let Some(options) = options {
         if !options.payload.is_null() {
-            if let Some(payload) = unsafe { *options.payload }.transmute_mut().extract() {
+            if let Some(payload) = unsafe { options.payload.as_mut() }
+                .unwrap()
```

Comment:
Not sure: what if `options.payload` points to `z_owned_bytes_t` in null state. I suppose `unwrap()` should panic in this case. Do I miss something?

Reply:
I check that the pointer is not null, that is the only guarantee I can get from C.

Reply:
It is an unwrap on ref, not on option, if let = Some(...) still checks for None